logstash-core/src/main/java/org/logstash/log/LoggerExt.java

```diff
@@ -31,10 +31,8 @@ public LoggerExt(final Ruby runtime, final RubyClass metaClass) {
     }
 
     @JRubyMethod(meta = true)
-    public static LoggerExt initialize(final ThreadContext context, final IRubyObject self, final IRubyObject loggerName) {
-        LoggerExt logger = new LoggerExt(context.runtime, (RubyClass) self);
-        logger.logger = LogManager.getLogger(loggerName.asJavaString());
-        return logger;
+    public static IRubyObject initialize(final ThreadContext context, final IRubyObject self, final IRubyObject configPath) {
```

Comment:
@danhermann you can just do:

```
 @JRubyMethod(meta = true, name = {"reconfigure", "initialize"})
```

on the annotation for `reconfigure` instead to define the alias. No need to repeat the signature `name` takes an array annotation argument :)